src/controllers/albumController.js

```diff
@@ -38,6 +81,14 @@ const updateAlbum = async (req, res) => {
 //will be user to let admin update price of several albums on the frontend
 
 const updatePriceOfAlbums = async (req, res) => {
+  /*
+     #swagger.summary = 'Update prices of albums passed in req.body'
+
```

Comment:
We should also define what the expected input is; as swagger-autogen isn't able to easily figure it out from this function's body:

```
	#swagger.parameters['albums to update prices'] = {
		in: 'body',
		description: 'Array of information to use to update album prices',
		required: true,
		type: 'array',
		schema: [{ id: "5f9d7b3b9d9d9d9d9d9d9d9d", price: 9.99 }]
 }
```